sktime/forecasting/compose/_reduce.py

```diff
@@ -1310,13 +1310,83 @@ def make_reduction(
     pooling="local",
     windows_identical=True,
 ):
-    """Make forecaster based on reduction to tabular or time-series regression.
+    r"""Make forecaster based on reduction to tabular or time-series regression.
```

Comment:
Just curious again, what is the raw string for?

Reply:
Same. see explanation below